### PR TITLE
Add recipe for nb2plots

### PR DESCRIPTION
**Pull request by CurtLH**

### CODE REVIEW

**Comment by CurtLH** [comment]
This one is ready for review @conda-forge/staged-recipes

File: recipes/nb2plots/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipython[notebook] >=3.0
```
Comment (isuruf) [inline recipes/nb2plots/meta.yaml]: Does this work?

Comment (ocefpaf) [reply recipes/nb2plots/meta.yaml]: Probably not. @CurtLH did conda-skeleton added that or you did it manually?

Comment (CurtLH) [reply recipes/nb2plots/meta.yaml]: I used conda-skeleton to start the recipe, and then modified it from there.  But `ipython[notebook] >=3.0` came from conda-skeleton.  

It is in the [requirements.txt](https://github.com/matthew-brett/nb2plots/blob/6d00a39ab29fb62d024a4456a772379ff71546b8/requirements.txt#L3)

Let me know and I can open a PR to remove it from the feedstock if needed.

**Pull request merged**